app/main.py

```diff
@@ -1,3 +1,59 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if isinstance(other, Distance):
+            return Distance(km=self.km + other.km)
+        raise TypeError("Operand must be of type 'Distance'")
+
+    def __iadd__(self, other):
+        if isinstance(other, Distance):
+            self.km += other.km
+            return self
+        raise TypeError("Operand must be of type 'Distance'")
```

Comment:
Similar to the `__add__` method, consider returning `NotImplemented` instead of raising a `TypeError` when the operand is not a `Distance` instance.